app/jsx/pages/BrowsePage.jsx

```diff
@@ -44,13 +43,13 @@ class BrowsePage extends PageBase
         // Campus-specific browse page
         <div>
           <Header2Comp type="campus" unitID={data.campusID} />
-          <Subheader2Comp unit={data.unit}
-                          campusID={data.campusID}
-                          campusName={data.campusName}
+          <Subheader2Comp unit={data.unit} logo={data.header.logo}
+                          campusID={data.header.campusID}
+                          campusName={data.header.campusName}
                           campuses={data.campuses} />
           <div className="c-navbar">
             {/* ToDo: Properly call header.nav_bar for unit type="campus" */}
-            <NavComp data={[{name: 'Open Access Policies', slug: ''}, {name: 'Journals', slug: '/' + data.campusID + '/journals'}, {name: 'Academic Units', slug: '/' + data.campusID + '/units'}]} />
+            <NavComp data={[{name: 'Open Access Policies', url: ''}, {name: 'Journals', url: '/' + data.header.campusID + '/journals'}, {name: 'Academic Units', url: '/' + data.header.campusID + '/units'}]} />
```

Comment:
Hmm, I'd like to verify with Justin and John that admins will not be able to add anything to the campus nav bar.